client/ayon_core/hosts/maya/api/pipeline.py

```diff
@@ -376,9 +377,12 @@ def _maya_iterate(iterator):
             yield iterator.thisNode()
             iterator.next()
 
-    ids = {AVALON_CONTAINER_ID,
-           # Backwards compatibility
-           "pyblish.mindbender.container"}
+    ids = {
+        AYON_CONTAINER_ID,
+        # Backwards compatibility
+        AVALON_CONTAINER_ID,
+        "pyblish.mindbender.container"
```

Comment:
This backwards compatibility is from the very early days of AVALON itself - not even OpenPype. I think it's time to remove it by now :)
```suggestion
        AVALON_CONTAINER_ID
```